agimus-hpp/Makefile

```diff
@@ -0,0 +1,32 @@
+# robotpkg Makefile for:	wip/agimus-hpp
+# Created:			Long Ha Thuc on Tue, 18 Aug 2020
+#
+ORG=			agimus
+NAME=			agimus-hpp
+VERSION=		1.0.0
+DOWNLOAD_LOC=   releases/download
+MASTER_SITE_AGIMUS=     https://github.com/agimus/
+DISTNAME=		${NAME}-${VERSION}
+
+MASTER_SITES=		${MASTER_SITE_AGIMUS:=${NAME}/${DOWNLOAD_LOC}/v${VERSION}/}
+MASTER_REPOSITORY=	${MASTER_REPOSITORY_GITHUB}${ORG}/${NAME}/${DOWNLOAD_LOC}/v${VERSION}
+LICENSE=		BSD-2-Clause
+
+CATEGORIES=		wip
+COMMENT=		Package to link HPP to the Agimus framework
+
+CMAKE_ARGS+=    -DPYTHON_STANDARD_LAYOUT=ON
```

Comment:
This agimus-hpp package uses the jrl-cmakemodule. Therefore, you can use:
```
include ../../devel/jrl-cmakemodules/Makefile.common
```
to setup everything you need: this `-DPYTHON_STANDARD_LAYOUT=ON` is automatically added there, among other features